docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -447,9 +447,10 @@ This section lists out the behavior for Cluster objects using `ClusterClass` in
     1. Sets the `version` field on the MD object from the `spec.topology.version`.
     1. Sets the `spec.template.spec.bootstrap` on the MD object from the `ClusterClass.spec.workers.machineDeployments[i].template.bootstrap.ref` field.
     1. Sets the `spec.template.spec.infrastructureRef` on the MD object from the `ClusterClass.spec.workers.machineDeployments[i].template.infrastructure.ref` field.
-    1. Generates the set of labels to be set on the MD object. The labels are additive to the class' labels list, and the value in the `spec.topology.workers.machineDeployments[i].labels` takes precedence over any set by the ClusterClass. Also, include the topology label stated below:
+    1. Generates the set of labels to be set on the MD object. The labels are additive to the class' labels list, and the value in the `spec.topology.workers.machineDeployments[i].labels` takes precedence over any set by the ClusterClass. Include the topology label as well as a label to track the name of the machine deployment:
```

Comment:
```suggestion
    1. Generates the set of labels to be set on the MD object. The labels are additive to the class' labels list, and the value in the `spec.topology.workers.machineDeployments[i].labels` takes precedence over any set by the ClusterClass. Include the topology label as well as a label to track the name of the MachineDeployment topology:
```
nit (not sure if that's the best way to express it but otherwise it really sounds like it's the name of the MachineDeployment, which it isn't)